client-tez/src/main/java/org/apache/tez/runtime/library/common/shuffle/impl/RssTezFetcher.java

```diff
@@ -133,7 +133,7 @@ public void copyFromRssServer() throws IOException {
         hasPendingData = false;
         uncompressedData = null;
       } else {
-        LOG.info("uncompressedData is null");
+        LOG.info("UncompressedData is null");
```

Comment:
I think this is not related with current issue. 

Anyway, let's reserve these changes.